server/database/models/books.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Books = sequelize.define('Books', {
+    id: DataTypes.INTEGER,
+    title: DataTypes.STRING,
+    description: DataTypes.TEXT,
+    isbn: DataTypes.INTEGER,
+    price: DataTypes.INTEGER,
+    yearPublished: DataTypes.INTEGER,
+    stock: DataTypes.INTEGER
+  }, {});
+  Books.associate = function(models) {
+    // associations can be defined here
+    Books.belongsTo(models.author, {
```

Comment:
It should be 'models.Author'. The author should be capitalized here.

Reply:
the author.js file in models folder is in small letters.

Reply:
In your Author definition, you used capital letters. That is how it should be here also.

Reply:
thats the authors database and it autogenerated. this is referencing the file in the models folder

Reply:
you need to rebase this branch

Reply:
it should be models.Authors

looking at your db the table name is Authors
![image](https://user-images.githubusercontent.com/44194416/58656548-647cdd00-8314-11e9-9a14-03adcd2f294e.png)

Reply:
good job overall make the changes so i can approve

Reply:
> thats the authors database and it autogenerated. this is referencing the file in the models folder
> 

On my local, I tried it using 'models.Author' and the association worked, so I don't think it's the name of the file that matters here. So for readability sake, I suggest you use 'Author', but that's up to you since it works with 'author' as well. Every other thing looks fine. Rebase the branch so I can approve. Cheers 👍.

Reply:
done